### PR TITLE
Can Parlar - Assignment5

### PR DESCRIPTION
**Pull request by canparlar**

### CODE REVIEW

File: assignment5.txt [hunk 1]
```diff
@@ -0,0 +1,10 @@
+Can Parlar
+1.A. It creates a copy and modifies the copy only. If you dont use return variableName command the variable will not return.
+
+1.B. It creates a reference of that varible in the other function and modifies it wile changing the original one too.
```
Comment (oldclesleycode) [inline assignment5.txt:4]: Right!

File: assignment5.txt [hunk 2]
```diff
@@ -0,0 +1,10 @@
+Can Parlar
+1.A. It creates a copy and modifies the copy only. If you dont use return variableName command the variable will not return.
```
Comment (oldclesleycode) [inline assignment5.txt:2]: Right!

File: assignment5.txt [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+
+1.B. It creates a reference of that varible in the other function and modifies it wile changing the original one too.
+
+2. It returns a integer or a datatype from a function to another, it terminates the code at the end and it just returns from a function to another.
```
Comment (oldclesleycode) [inline assignment5.txt:6]: Right, it returns control and a value back to the calling function

File: assignment5.txt [hunk 4]
```diff
@@ -0,0 +1,10 @@
...
+
+3. It enables reuse without rewriting the function again and makes the code cleaner.
```
Comment (oldclesleycode) [inline assignment5.txt:8]: right! also makes debugging less painful

File: palindrome.c [hunk 5]
```diff
@@ -0,0 +1,53 @@
+/* Can Parlar - scans a string and checks if it is a palindrome word or not
+*/
+#include <stdio.h>
```
Comment (oldclesleycode) [inline palindrome.c:3]: your code unfortunately doesn't handle many of the test cases: 
1. empty string - this is a palindrome. 
2. single letters - "a" or "d" or one letter of the alphabet is a palindrome.
3. numbers - 121 is a palindrome
4. capitals - "Hannah" is a palindrome

You should  thoroughly test your code to get at these test and edge cases.

File: three.c [hunk 6]
```diff
@@ -0,0 +1,66 @@
+/* Can Parlar - scans 3 numbers and checks if they can make a valid triangleand what will be the type of the triangle */
+#include <stdio.h>
```
Comment (oldclesleycode) [inline three.c:2]: you submitted with three warnings, which were due to parentheses of the boolean expressions

Comment (oldclesleycode) [inline three.c:2]: when an invalid triangle was input, it still evaluated a type of triangle, which shouldn't have been the case.
